src/connectionconfig/connectionDialogWebviewController.ts

```diff
@@ -994,9 +1009,17 @@ export class ConnectionDialogWebviewController extends FormWebviewController<
                     return await this.handleConnectionErrorCodes(connectionInfo, state);
                 }
             } catch (error) {
-                this.state.formError = getErrorMessage(error);
+                this.state.formMessage = { message: getErrorMessage(error) };
                 this.state.connectionStatus = ApiStatus.Error;
 
+                if (
```

Comment:
should this go into handleConnectionErrrorCodes method? Also do you want to handle this error on the connectionManager level? 
https://github.com/microsoft/vscode-mssql/blob/097b79d2d077f7615bf3f5071c872924ddeff7bd/src/controllers/connectionManager.ts#L2001

Reply:
Maybe we can create a new error type and handle the error type in both connection manager and connection dialog webview controller.